.circleci/config.yml

```diff
@@ -20,6 +20,8 @@ jobs:
             INSTALL_SMALL_PYTHON_DEPS: true
             INSTALL_LARGE_PYTHON_DEPS: true
           command: |
+            pip --version
+            pip install --upgrade 'pip<20.3'
```

Comment:
do we want `pip!=20.3` so that we pick up a newer pip once this problem is fixed?

Reply:
makes sense. I'll make the change

Reply:
done!